Dfe.Data.SearchPrototype/Web/Tests/Web.Integration.UI.Tests/DependencyInjection/UIServices.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Dfe.Data.SearchPrototype.Web.Tests.Shared.Pages.Components;
+using Dfe.Data.SearchPrototype.Web.Tests.Shared.Pages;
+using Microsoft.Extensions.DependencyInjection;
+using Dfe.Testing.Pages;
+
+namespace Dfe.Data.SearchPrototype.Web.Tests.Web.Integration.UI.Tests.DependencyInjection;
+
+internal sealed class UIServices
+{
+
+    private static readonly UIServices _instance = new();
+    private readonly IServiceProvider _serviceProvider;
+    static UIServices()
+    {
+    }
+    
+    private UIServices()
+    {
+        IServiceCollection services = new ServiceCollection()
```

Comment:
There's an internal method now to share application components `AddPages` so we don't need to add all types in each project.